public/dom/index.js

```diff
@@ -0,0 +1,17 @@
+/* eslint no-var: 0 */
+/* eslint prefer-arrow-callback: 0 */
+/* eslint func-names: 0 */
+/* eslint wrap-iife: 0 */
+
+function clientRequest(method, url, body, callback) {
+  var xhr = new XMLHttpRequest();
+  xhr.addEventListener('load', function () {
+    if (xhr.readyState === 4 && xhr.status === 200) {
+      callback(JSON.parse(xhr.responseText));
```

Comment:
JSON.parse() may throw an exception. We can refactor this later to include a try catch block

Reply:
Added this as a comment to issue #48